Cargo.toml

```diff
@@ -13,14 +13,12 @@ exclude = [
     "no-std-check",
 ]
 
-# Our tests do actual crypo and lots of work, the tradeoff for -O1 is well worth it.
-# Ideally we would only do this in profile.test, but profile.test only applies to
-# the test binary, not dependencies, which means most of the critical code still
-# gets compiled as -O0. See
+# Our tests do actual crypo and lots of work, the tradeoff for -O1 is well
```

Comment:
> "[...] the tradeoff for -O1 is well worth it."

Wait, I now thought the `test` and `dev` profiles are disjunct, and here we're only enabling `-O1` for the latter?

Also: `s/crypo/crypto`